### ISSUE TITLE
Stock: gui demo isn't working on IE

### ISSUE THREAD

**Issue by raf18seb** [highcharts/highcharts#16129]
`Element.classList` isn't working on IE.

```js
const buttonWrapper = button.parentNode,
    buttonWrapperClass = buttonWrapper.classList.value;
```

https://github.com/highcharts/highcharts/blob/49a925eb98237f8d303a3fe90431ea2a8bf2e9b3/ts/Stock/StockToolsGui.ts#L1555

#### Live demo with steps to reproduce
https://www.highcharts.com/demo/stock/stock-tools-gui

#### Product version
Highcharts Stock 9.1.2

#### Affected browser(s)
IE

**Comment by khlieng** [comment]
Duplicate of #16087, fixed by #16088 😉

**Comment by raf18seb** [comment]
I was sure I saw it somewhere, just couldn't find where! Hmm
